src/Workspaces/Core/Portable/CodeActions/CodeAction.cs

```diff
@@ -223,73 +251,133 @@ public Task<ImmutableArray<CodeActionOperation>> GetPreviewOperationsAsync(Cance
         }
 
         /// <summary>
-        /// Override this method if you want to implement a <see cref="CodeAction"/> subclass that includes custom <see cref="CodeActionOperation"/>'s.
+        /// Override this method if you want to implement a <see cref="CodeAction"/> subclass that includes custom <see
+        /// cref="CodeActionOperation"/>'s.
```

Comment:
Should documentation be added that indicates the other method is now preferable?